
import logging
logging.exception("Hello World")
